sci-libs/cantera/metadata.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="person">
+    <email>torokhov_s_a@mail.ru</email>
+    <name>Sergey Torokhov</name>
+  </maintainer>
+  <maintainer type="project">
+    <email>proxy-maint@gentoo.org</email>
+    <name>Proxy Maintainers</name>
+  </maintainer>
+  <longdescription>
+    Cantera is an open-source suite of object-oriented software tools
+    for problems involving chemical kinetics, thermodynamics, and/or transport processes.
+  </longdescription>
+  <use>
+    <flag name="cpp">Install C++ API libraries, headers and samples</flag>
+    <flag name="cti">Install CTI tools (ck2cti, ctml_writer) for conversion of Chemkin files to Cantera format</flag>
+    <flag name="static-libs">Install static version of C++ dynamic library as well</flag>
```

Comment:
You only need to describe flags that are local to this package. Don't describe stuff that's exactly equal to the global definition.

Reply:
I will rename `cpp` to `cxx` and remove it from metadata.xml.

Could I leave the description of `static-libs` description as the `fortran` interface library is provided only by static library? So I initially wanted to handle by `static-libs` flag only the presence of C++ static library without affecting the Fortran static library. Maybe it worth additionally to notice about this fact here.

Reply:
Done partially: `static-libs` is still within `metadata.xml` because it has some different behavior in comparison with global use-flag. It's local descriptions is changed to 

> Install static version of C++ dynamic library as well. This flag doesn't affect Fortran static library.

Is it ok in this case ?